how-to/integrate-with-rss/public/manifest.fin.json

```diff
@@ -0,0 +1,81 @@
+{
+	"devtools_port": 9090,
+	"licenseKey": "openfin-demo-license-key",
+	"runtime": {
+		"arguments": "--v=1 --inspect",
+		"version": "26.102.71.5"
+	},
+	"platform": {
+		"uuid": "integrate-with-rss",
+		"icon": "http://localhost:8080/favicon.ico",
+		"autoShow": false,
+		"providerUrl": "http://localhost:8080/platform/provider.html",
+		"preventQuitOnLastWindowClosed": true,
+		"permissions": {
+			"System": {
+				"openUrlWithBrowser": {
+					"enabled": true,
+					"protocols": ["mailto"]
+				}
+			}
+		},
+		"defaultWindowOptions": {
+			"permissions": {
+				"System": {
+					"openUrlWithBrowser": {
+						"enabled": true,
+						"protocols": ["mailto"]
+					}
+				}
+			}
+		}
+	},
+	"snapshot": {
+		"windows": []
+	},
+	"shortcut": {
+		"company": "OpenFin",
+		"description": "An way of showing examples of what OpenFin can do.",
+		"icon": "http://localhost:8080/favicon.ico",
+		"name": "Integrate with RSS - v9.0",
+		"target": ["desktop", "start-menu"]
+	},
+	"customSettings": {
+		"homeProvider": {
+			"id": "integrate-with-rss",
+			"title": "Integrate with RSS",
+			"icon": "http://localhost:8080/favicon.ico"
+		},
+		"integrationProvider": {
+			"integrations": [
+				{
+					"id": "rss",
+					"icon": "http://localhost:8080/favicon.ico",
+					"title": "RSS Provider",
+					"enabled": true,
+					"moduleUrl": "http://localhost:8080/js/integrations/rss.bundle.js",
+					"data": {
+						"rootUrl": "http://localhost:8080/",
```

Comment:
Do we need proxy address?

Shouldn't this be rss url (which happens to point to a proxy)?

Reply:
The data is sent to the proxy as a POST, in case you want to send additional headers etc to the endpoint - I could make it purely a get